src/CMakeLists.txt

```diff
@@ -122,16 +122,10 @@ if(BUILD_CLIENT)
 	endif()
 endif()
 
-
-option(ENABLE_FREETYPE "Enable FreeType2 (TrueType fonts and basic unicode support)" TRUE)
 set(USE_FREETYPE FALSE)
```

Comment:
shouldn't this be removed?

Reply:
It could be removed by replacing `USE_FREETYPE` with `BUILD_CLIENT` everywhere but it was a bit easier not do this.

Reply:
According to my search, you've already removed it from everywhere in the source, so that leaves just a few places in CMake to remove it (ignoring `minetest.exe`, of course):

![image](https://user-images.githubusercontent.com/31123645/148455373-8270927d-f524-4e20-88de-dd8725984781.png)

Since it's so few, I would definitely say remove it.

Reply:
Okay.